scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScannerTokens.scala

```diff
@@ -929,6 +955,21 @@ object ScannerTokens {
     loop(str.length - 1, 0)
   }
 
+  // return None if single line, or Some(all indents are at least minIndent)
+  private def multilineCommentMinIndent(t: Comment, minIndent: Int): Option[Boolean] = {
```

Comment:
Should we add some tests for multiline comments? Should we even accept multiline comments between operators, don't they count as empty line?

Reply:
1. we can add tests, yes; i will revise.
2. i used scastie to test out the behaviour; handling of comments is not mentioned on the page i linked on line 712 above, unfortunately.

Reply:
just a couple of test to make sure we don't regress in the future would be awesome.

Reply:
done. also, after additional scastie testing, changed how the infix argument is validated.

also, added a new `EOL` token called `InfixLF` so that we don't need to check this arg a second time in `ScalametaParser`.